widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadPositionHolder.java

```diff
@@ -0,0 +1,21 @@
+package com.glia.widgets.view.head;
```

Comment:
are ChatHead and ChatBubble the same things? If so then stick to one naming.

Reply:
chat head should be used